packages/components/icon/src/icon.css

```diff
@@ -98,3 +98,12 @@ svg,
 :host([type='suggested']) svg {
 	color: var(--ts-color-purple);
 }
+
+:host([circular]) {
+	border-radius: 40em;
```

Comment:
What is the number ? 40em

Reply:
Just a large number, Habit for making rounded corners for non square shapes, not necessary here since there are square already